docker-compose.yaml

```diff
@@ -65,6 +75,8 @@ services:
       # https://docs.gatewayd.io/using-gatewayd/configuration#environment-variables
       - GATEWAYD_CLIENTS_DEFAULT_WRITES_ADDRESS=write-postgres:5432
       - GATEWAYD_CLIENTS_DEFAULT_READS_ADDRESS=read-postgres:5432
+      # Use the connection string set in install_plugins
+      - POSTGRES_URL=${POSTGRES_URL}
```

Comment:
This is not used anywhere in the `setup.sh` script or config files and seems unnecessary.